src/modules/roc_sndio/target_sox/roc_sndio/player.h

```diff
@@ -60,11 +58,23 @@ class Player : public core::Thread {
     //!  Should be called once before calling start().
     bool open(const char* name, const char* type = NULL);
 
+    //! Start reading samples.
+    //!
+    //! @b Parameters
+    //!  - @p input is used to read samples.
+    void start(pipeline::IReceiver* input);
+
     //! Stop thread.
     //! @remarks
     //!  Can be called from any thread.
     void stop();
 
+    //! Get sample rate
+    //!
```

Comment:
It's worth to mention how this sample rate is obtained.

Reply:
Also, we need a test for this feature.

Reply:
Fixed

Reply:
Fixed